services/sensu_webhook.rb

```diff
@@ -0,0 +1,22 @@
+class SensuWebhook
+  REDIS_KEY_PREFIX = 'sensu'
+
+  def initialize(payload)
+    @payload = payload
+  end
+
+  def process
+    service_id = @payload['key']
+    redis_key = "#{REDIS_KEY_PREFIX}/#{service_id}"
```

Comment:
My knowledge of redis isn't great, but for other services, aren't we using a colon as a divider not a forward slash?

https://github.com/ministryofjustice/2nd-line-support/blob/master/models/traffic_spike.rb#L8